src/app/components/SearchResults/Filters.jsx

```diff
@@ -83,38 +81,81 @@ const Filters = ({
     })
     .slice(0, 10);
 
+  // beginning to prepare for not-js
+  const onSubmit = (e) => {
+    e.preventDefault();
+    e.stopPropagation();
+
+    doSearchWithFilters(filtersArray);
+  };
+
+  const onChangeYears = (yearsFilter) => {
+    const currentYearsFilter = {
+      field: 'years',
+      value: yearsFilter,
+    };
+    const matchIndex = filtersArray.findIndex(filter => filter.field === 'years');
+    if (matchIndex === -1) {
+      filtersArray.push(currentYearsFilter);
+    } else if (matchIndex > -1) {
+      filtersArray[matchIndex] = currentYearsFilter;
+    }
+  };
+
   if (data && data.facets && data.hits && data.hits.hits && data.hits.hits.length > 0) {
     return (
-      <form className="filters usa-form">
+      <form
+        className="filters usa-form"
+        action="/search"
+        onSubmit={onSubmit}
+      >
+        <input
+          type="hidden"
+          name="query"
+          value={searchQuery.query}
+        />
+        <input
+          type="hidden"
+          name="field"
+          value={searchQuery.field}
+        />
         <div className="filters-header">Filter data</div>
         {Object.keys(filtersLabels).map(field => (
           <fieldset
             key={field}
             className="filters-box usa-fieldset"
           >
-            {data.facets[field] && <legend className="filters-box-header">{filtersLabels[field]}</legend>}
-            {prepareFilters(data.facets[field], field).map(facet => (
-              <div
-                className="usa-checkbox"
-                key={`facet-${field}-${facet.value}`}
-              >
-                <input
-                  className="usa-checkbox__input"
-                  id={`facet-${field}-${facet.value}`}
-                  type="checkbox"
-                  name={facet.value}
-                  onChange={e => onChangeCheckbox(e, field, facet.value)}
-                  checked={isFilterChecked(field, facet.value)}
-                />
-                <label
-                  className="usa-checkbox__label"
-                  htmlFor={`facet-${field}-${facet.value}`}
+            <legend className="filters-box-header">{filtersLabels[field]}</legend>
```

Comment:
This was giving me a syntax error that I fixed by editing this into separate rows like:
```
<legend className="filters-box-header">
  {filtersLabels[field]}
</legend>
```